lib/core/test/data/Cardano/Wallet/Api/ApiBlockReference.json

```diff
@@ -1,85 +1,95 @@
 {
-    "seed": -2366744487741549495,
+    "seed": -971028748466080651,
     "samples": [
         {
             "height": {
-                "quantity": 18267,
+                "quantity": 20557,
                 "unit": "block"
             },
-            "epoch_number": 16540,
-            "slot_number": 15227
+            "epoch_number": 31772,
+            "slot": 15286302,
+            "slot_number": 9159
         },
         {
             "height": {
-                "quantity": 7566,
+                "quantity": 21832,
                 "unit": "block"
             },
-            "epoch_number": 408,
-            "slot_number": 28796
+            "epoch_number": 11776,
+            "slot": 3700914,
+            "slot_number": 32043
```

Comment:
Looks a bit weird with `slot` vs `slot_number`, but probably sane for compatibility 👍

Reply:
Changed to `absolute_slot_number`.